provisioning/e2e/_provisioning_e2e.js

```diff
@@ -53,7 +53,7 @@ describe('IoT Provisioning', function() {
         x509: {
           clientCertificates: {
             primary: {
-              certificate: x509.cert
+              certificate: new Buffer(x509.cert).toString('base64')
```

Comment:
>new Buffer(x509.cert).toString('base64') [](start = 27, length = 40)

Really?  I'm pretty sure that this would be a pem or a cert and didn't need to be bin64'd again.